src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -145,7 +148,7 @@ function useChatUtilsHook(): UseChatUtilsHook {
   const goToMoveChat = (channel: ChannelList) => {
     setChat({
       ...chat,
-      selectedChannel: channel
+      selectedChannel: channel as unknown as null
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, assigning `null` to `selectedChannel` here could cause issues if other parts of your code expect this property to always be a `ChannelList` object. Ensure to add null checks wherever `selectedChannel` is used.
```diff
- selectedChannel: channel as unknown as null
+ selectedChannel: null
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->